release/release_after_kokoro.sh

```diff
@@ -201,15 +201,16 @@ categoryId=com.google.cloud.tools.eclipse.category
 featureId=com.google.cloud.tools.eclipse.suite.e45.feature.feature.group
 copyrightText="Copyright 2016, 2017 Google Inc."
 licenseUri=https://www.apache.org/licenses/LICENSE-2.0
-licensesText="Cloud Tools for Eclipse is made available under the Apache\
+licenseText="Cloud Tools for Eclipse is made available under the Apache\
  License, Version 2.0. Please visit the following URL for details:\
  https://www.apache.org/licenses/LICENSE-2.0"
+categoryXPathExpr="/repository[@name='$repoName']/units/unit[@id='$categoryId']"
 
-valid=$(unzip -p gcp-repo/target/repository/content.jar \
-  | xml sel -t -m "/repository[@name='$repoName']/units/unit[@id='$categoryId']" \
-     -v "normalize-space(copyright)='$copyrightText' \
-         and normalize-space(licenses[@size=1]/license[@uri='$licenseUri'])='$licensesText' \
-         and requires[@size='1']/required/@name='$featureId'")
+valid=$(unzip -p $LOCAL_REPO/gcp-repo/target/repository/content.jar \
```

Comment:
`$LOCAL_REPO` is `$WORK_DIR/repository`. I don't think it will have `gcp-repo`. Should it be `$LOCAL_REPO/content.jar`?